xAPI-About.md

```diff
@@ -643,6 +634,9 @@ and "category" contextActivities.
     "contextActivities":{  
       "category":[  
         {  
+       	  "id":"https://w3id.org/xapi/cmi5"
```

Comment:
This should not be added. The cmi5 context category is already present:

> https://w3id.org/xapi/cmi5/context/categories/cmi5

Reply:
Perhaps we should remove this example all together? I don't like this as an example for other people to copy. To be consistent the base profile IRI should be https://w3id.org/xapi/cmi5. @andyjohnson andy word on updating this in the cmi5 spec? 

I propose we have a more simpler example than cmi5.

Reply:
The base profile ID can be https://w3id.org/xapi/cmi5, but what goes in the category context activities is not the base profile ID.

Reply:
So that Could be the version of the profile, correct? For example, if I were to update this example it could be https://w3id.org/xapi/cmi5/v1 instead of the base?

Reply:
It shouldn't be https://w3id.org/xapi/cmi5/v1 because the current version of cmi5 uses https://w3id.org/xapi/cmi5/context/categories/cmi5 to indicate conformance to cmi5, making that the right URI for the first version of the cmi5 Profile.

Reply:
I thought the context category id should be the profile IRI or a specific version of the profile? This strange id that cmi5 came up with for the context category is a bad example/practice if we want consistency in support things like using content negotiation and pattern matching to dereference profile IRIs (for the JSON statement templates, concepts, and the correct version of the profile).

Reply:
The context category ID must not be the profile IRI; it must be the IRI of a specific version of the profile. Since cmi5 has an already-picked context category ID, to use the profile spec with existing cmi5 practice requires that be the IRI of the current version of cmi5. Content negotiation has nothing at all to do with the form of the URI, and people should not be using pattern matching to dereference profile IRIs -- IRIs should be treated as opaque identifiers.

If we were coming up with a cmi5 profile version IRI from scratch we would likely pick a different IRI. We're not, so we need to go with the one we have, but that's fine.

Reply:
Actually, pattern matching is a smart and efficient way to handle all sorts of stuff for content negotiation. See this, https://www.w3.org/TR/swbp-vocab-pub/#negotiation

Anyhow, all of the profile IRIs are set up to handle requests for JSON, JSON-LD, Turtle, and HTML. It's no big deal...it's just a little more of a pain to deal with "one-off's" but as you pointed out if we were starting from scratch we could enforce this better I suppose. 

My point in all this is that it sets a bad example for others to copy if the only profile that has a different looking IRI for use in context category is the one we are using as an example in the spec. I'm probably just worrying about something that will just be addressed during curation of the profiles anyway. 

I'll close this PR and start a new one.

Reply:
The link you gave is to content negotiation, which is great, but has nothing to do with the form of the IRI, and none of that has anything to do with pattern matching for dereferencing.

I'm not especially worried about being a bad example, but even if it is, it is not avoidable without deciding not to do a Profiles document for the current version of cmi5.

Reply:
hmm.. perhaps either I'm not explaining it well. Here's what we are doing
for xapi profile IRIs:

https://github.com/perma-id/w3id.org/blob/master/xapi/.htaccess

To avoid writing unique rules for every profile you can create regex
pattern matching rules.